ScriptSpecification.md

```diff
@@ -124,64 +126,69 @@ Everything after the number will be ignored for compilation and follows no speci
-| 135 | Compares if one literal value or variable is smaller than another literal value or variable and sets to another variable.<br>```$local1 = 1 < 2;```<br>```$local1 = $local2 < 2;```<br>```$local1 = $local3 < $local4;``` |
+| 132 | Compares if one literal value or variable is greater or equal to another literal value or variable and sets to another variable. Lexographically compares strings. <br>```$local1 = 1 >= 2;```<br>```$local1 = $local2 >= 2;```<br>```$local1 = $local3 >= $local4;``` |
+| 133 | Compares if one literal value or variable is smaller or equal to another literal value or variable and sets to another variable. Lexographically compares strings. <br>```$local1 = 1 <= 2;```<br>```$local1 = $local2 <= 2;```<br>```$local1 = $local3 <= $local4;``` |
+| 134 | Compares if one literal value or variable is greater than another literal value or variable and sets to another variable. Lexographically compares strings. <br>```$local1 = 1 > 2;```<br>```$local1 = $local2 > 2;```<br>```$local1 = $local3 > $local4;``` |
+| 135 | Compares if one literal value or variable is smaller than another literal value or variable and sets to another variable. Lexographically compares strings. <br>```$local1 = 1 < 2;```<br>```$local1 = $local2 < 2;```<br>```$local1 = $local3 < $local4;``` |
 
 #### Booleans
 | Type | Description |
 | - | - |
-| 120 | Converts a literal value or variable to boolean and negate it.<br>```$local1 = not $local2;``` |
-| 121 | Converts two literal values or variables to boolean and boolean and them.<br>```$local1 = true && true;```<br>```$local1 = $local2 && true;```<br>```$local1 = $local2 && $local3;``` |
-| 122 | Converts two literal values or variables to boolean and boolean or them.<br>```$local1 = true \|\| true;```<br>```$local1 = $local2 \|\| true;```<br>```$local1 = $local2 \|\| $local3;``` |
+| 120 | Coerces a literal value or variable to boolean and negates it.<br>```$local1 = not $local2;``` |
+| 121 | Coerces two literal values or variables to boolean and runs a logical AND operation on them.<br>```$local1 = true && true;```<br>```$local1 = $local2 && true;```<br>```$local1 = $local2 && $local3;``` |
+| 122 | Coerces two literal values or variables to boolean and runs a logical OR operation on them.<br>```$local1 = true \|\| true;```<br>```$local1 = $local2 \|\| true;```<br>```$local1 = $local2 \|\| $local3;``` |
 
 #### Strings
 | Type | Description |
 | - | - |
 | 500 | Was originally used to log a message in developement. Is a no-op in published games.<br>```$local1 = log("This is a message.");``` |
 | 501 | Formats a string with placeholder values.<br>```$local1 = format("Formatted message %s", $local2);``` |
-| 503 | Get a substring from another string.<br>```$local1 = substring("This message", 5);``` |
+| 502 | Converts a UTF-8 encoded string into a UTF-16 (wide) string.<br>```$local1 = utf8_to_wide("hi");``` |
```

Comment:
To stay consistent with the start of the sentence. "[...] a UTF-16 (wide) encoded string."